docsrc/contents/smt.rst

```diff
@@ -476,6 +475,7 @@ Comparing GMPEs
         Z1 = -999
         Z25 = -999
         up_or_down_dip = 1 # 1 = up-dip, 0 = down-dip
+        region = 0 # get region specific z1pt0 and zpt50 ('Global' or 'Japan') 
```

Comment:
here and at line 156: should this now be indicating "Global" or "Japan" based on the changes in `utils_gmpes.py`?